torch/functional.py

```diff
@@ -357,6 +381,10 @@ def meshgrid(*tensors):
             tensor([[4, 5, 6],
                     [4, 5, 6],
                     [4, 5, 6]])
+            >>> torch.equal(torch.cat(tuple(torch.dstack([grid_x, grid_y]))),
```

Comment:
A comment before the first example and a comment explaining what this equality demonstrates might be helpful

Reply:
good idea!